src/fixmath/fix16_trig.c

```diff
@@ -16,13 +16,13 @@ static fix16_t _fix16_atan_cache_value[4096] = { 0 };
 
 fix16_t fix16_sin_parabola(fix16_t inAngle)
 {
-	fix16_t abs_inAngle, abs_retval, retval;
+	fix16_t abs_inAngle, retval;
```

Comment:
`abs_` variable name prefix?

Reply:
That's not mine, but fixmath's. I've moved abs_retval past ifdef so that gcc would not yell at me that I have unused variable.